src/poggit/release/submit/PluginSubmission.php

```diff
@@ -402,8 +406,7 @@ public function save(): int {
             }
             $releaseId = Mysql::query("INSERT INTO releases
             (name, shortDesc, artifact, projectId, buildId, version, description, icon, changelog, license, licenseRes, flags, state, parent_releaseId)
-     VALUES (?   , ?        , ?       , ?        , ?      , ?      , ?          , ?   , ?        , ?      , ?         , ?    , ?    , ?)", str_replace(["\n", " "], "", "
-             s     s          i         i          i        s        i            s     i          s        i           i      i      i"),
+     VALUES (?   , ?        , ?       , ?        , ?      , ?      , ?          , ?   , ?        , ?      , ?         , ?    , ?    , ?)", str_replace(["\n", " "], "", "ssiiisisisiiii"),
```

Comment:
what's up here?

Reply:
Causes errors, one sec ill get a ref

Reply:
`Jan 15 19:39:44.106 [69edf7d14369f29b] Level 0: mysqli_stmt::bind_param(): Number of elements in type definition string doesn't match number of bind variables in C:\Users\Jack\Documents\GitHub\poggit\src\poggit\utils\internet\Mysql.php#123
#0 [internal function]: poggit\error_handler(2, 'mysqli_stmt::bi...', 'C:\\Users\\Jack\\D...', 123, Array)
#1 C:\Users\Jack\Documents\GitHub\poggit\src\poggit\utils\internet\Mysql.php(123): mysqli_stmt->bind_param('\rssiiisisisiiii', 'TestPlugin', 'Test plugin for...', 6, 1, 19201, '1.0.0', 4, NULL, false, 'custom', 5, 8, 2, NULL)
#2 C:\Users\Jack\Documents\GitHub\poggit\src\poggit\release\submit\PluginSubmission.php(409): poggit\utils\internet\Mysql::query('INSERT INTO rel...', '\rssiiisisisiiii', 'TestPlugin', 'Test plugin for...', 6, 1, 19201, '1.0.0', 4, NULL, false, 'custom', 5, 8, 2, NULL)
#3 C:\Users\Jack\Documents\GitHub\poggit\src\poggit\release\submit\NewSubmitAjax.php(67): poggit\release\submit\PluginSubmission->save()
#4 C:\Users\Jack\Documents\GitHub\poggit\src\poggit\module\AjaxModule.php(44): poggit\release\submit\NewSubmitAjax->impl()
#5 C:\Users\Jack\Documents\GitHub\poggit\src\poggit\Meta.php(212): poggit\module\AjaxModule->output()
#6 C:\Users\Jack\Documents\GitHub\poggit\entry.php(50): poggit\Meta::execute('submit.new.ajax')
#7 C:\xampp\htdocs\index.php(23): include('C:\\Users\\Jack\\D...')
#8 {main}`